mobile/lib/common/domain/model.dart

```diff
@@ -64,3 +64,49 @@ class Amount {
     return formatSats(this);
   }
 }
+
+class Usd {
```

Comment:
🔧 It looks like this component is not strictly useful for usd but rather for any fiat currency. How about renaming it to Fiat?

Reply:
Let's deal with this once we have other currencies than USD :)